code/enumerable/index-find-vs-find-index.rb

```diff
@@ -0,0 +1,17 @@
+require 'benchmark/ips'
+
+ARRAY = (1..100).to_a
+
+def slow
+  ARRAY.index(ARRAY.find { |i| i % 5 == 0 && i % 7 == 0 })
+end
+
+def fast
+  ARRAY.find_index { |i| i % 5 == 0 && i % 7 == 0 }
```

Comment:
why not use some fixed number instead?
`%` adds some impact on results, so I believe `ARRAY.find_index { 42 }` might show a bit more correct benchmark numbers